src/Common/Core/Impl/Security/Credentials.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Net;
+using System.Security;
+
+namespace Microsoft.Common.Core.Security {
+    public sealed class Credentials : ICredentials {
+        // Although NetworkCredential does support SecureString, it still exposes
```

Comment:
Yes. However, password is visible as plain text in a debugger since debugger evaluates properties and when `Password` is evaulated, plain text is returned. This is debugger-only scenario but ppl on the Net recommend to hide it since one may expose password to a co-worked when pair-debugging. I.e. this is mostly for us :-)